app/queries/drug_stocks_query.rb

```diff
@@ -76,7 +76,7 @@ def drug_consumption_report
 
     custom_drug_category_order = CountryConfig.current.fetch(:custom_drug_category_order, [])
 
-    if custom_drug_category_order.to_set == drugs_by_category.keys.to_set
+    if (drugs_by_category.keys - custom_drug_category_order).empty?
```

Comment:
Apply the ordering even if the categories are a subset of the custom order list. This way, West Bengal can track DM drugs and leverage this order AND states that are not tracking DM drugs will still be able to leverage the hypertension drugs order.